src/containers/SplashScreen.js

```diff
@@ -4,14 +4,17 @@ import { connect } from 'react-redux';
 function SplashScreen(props) {
 	
 	return (
-		<div className="text-center animated fadeIn w-100" style={{margin: "20% auto 0"}}>
+		<div className="text-center animated fadeIn w-100">
 			{props.brand_image != null ?
 			<img
 				src={props.brand_image.full}
 				alt={props.title}
-				style={{maxWidth: "38%"}}
-			/> : <h1>Loading ...</h1>}
-			<div className="progress mt-3" style={{maxWidth: "38%", margin: "20px auto 0 auto"}}>
+				style={{width: "38%", position: "absolute", bottom: '60%', left: '50%', transform: 'translate(-50%, 0)'}}
```

Comment:
@Aringoaway Is it possible to add those style to splashscreen.scss (or some other general stylesheet e.g. `styles/index.scss`) ? Or does it have to be styled directly in the component due to pending of the scss files on application initialization?